backend/src/main/webapp/src/pages/ApplicationPage.tsx

```diff
@@ -16,6 +25,27 @@ export const ApplicationPage: React.FunctionComponent = () => {
     const applicationTypesQuery = useApplicationTypes(applicationId);
     const columns = [ 'Event Type', 'Event Type Id', 'Description' ];
 
+    const newEvent = useCreateEventType();
+    const [ id ] = React.useState<string | undefined>();
+    const [ displayName, setDisplayName ] = React.useState<string | undefined>();
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
We should remove this. I think we are using the variables below it. (or remove the unused variables)